python_scripts/api_requests.py

```diff
@@ -132,19 +183,25 @@ def get_bookings_df(bookings):
         bookings_df, bookings_metadata
     )
 
+    # replace nan to empy string
+    bookings_df.replace('nan', '', inplace=True)
+    
     return bookings_df
 
 
-def get_locations_df(locations):
-    locations_df = pd.io.json.json_normalize(locations)
-    renames = read_json("metadata/locations_renames.json")
+def get_locations_df(locations, db_version, env):
+    locations_df = pd.json_normalize(locations)
+    renames = read_json(f"metadata//{db_version}/{env}/locations_renames.json")
     locations_df = locations_df[renames.keys()].rename(columns=renames)
-    locations_metadata = read_json("metadata/locations.json")
+    locations_metadata = read_json(f"metadata/{db_version}/{env}/locations.json")
```

Comment:
As with bookings df: 
To handle db version and environment.
Also removed `io.json` as this was depreceated in later version of pandas